Patches/Kemomimihouse/ThingDefs_Races/Race_Animals.xml

```diff
@@ -0,0 +1,589 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+  <Operation Class="PatchOperationSequence">
+	<operations>
+	  <li Class="PatchOperationFindMod">
+			
+		<mods><li>kemomimihouse</li></mods>
+			
+		<match Class="PatchOperationSequence">
+		<operations>
+			
+			<li Class="PatchOperationAddModExtension">
+				<xpath>/Defs/ThingDef[
+				defName="akame" or
+				defName="aome" or
+				defName="hana" or
+				defName="kon" or
+				defName="kuro" or
+				defName="miko" or
+				defName="siro" or
+				defName="yuki"
+				]</xpath>
+				<value>
+					<li Class="CombatExtended.RacePropertiesExtensionCE">
+						<bodyShape>Humanoid</bodyShape>
+					</li>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationAdd">
+				<xpath>/Defs/ThingDef[
+				defName="akame" or
+				defName="aome" or
+				defName="hana" or
+				defName="kon" or
+				defName="kuro" or
+				defName="miko" or
+				defName="siro" or
+				defName="yuki"
+				]/comps</xpath>
+				<value>
+					<li Class="CombatExtended.CompProperties_Inventory" />
+				</value>
+			</li>
+
+			
+			<li Class="PatchOperationAdd">
+			 <xpath>/Defs/ThingDef[defName="akame" or
+				defName="aome" or
+				defName="hana" or
+				defName="kon" or
+				defName="kuro" or
+				defName="miko" or
+				defName="siro" or
+				defName="yuki"
+				]/statBases</xpath>
+				  <value>
+					<Suppressability>0</Suppressability>

```

Comment:
Unnecessary because they're animals and don't have the associated comp, but harmless.